tofu/imas2tofu/_core.py

```diff
@@ -2251,6 +2299,18 @@ def _get_t0(self, t0=None):
 
     def to_Config(self, Name=None, occ=None,
                   description_2d=None, mobile=None, plot=True):
+        """ Export the content of wall ids as a tofu Config object
+
+        Choose the occurence (occ), and index (description_2d, cf. dd_doc) to
```

Comment:
what is `dd_doc` ?

Reply:
It is an IMAS-specific documentation
Users of IMAS will know it, others won't, but anyway, this sub-package imas2tofu is to be used by IMAS users only, so they will know  ;-)

Basically, to get the documentation on IMAS on the ITER clusters for example, you typically type dd_doc in a terminal (with the ritgh modules pre-loaded), and a web browser opens ndirectly on an IMAS documentation page